items/models.py

```diff
@@ -2,13 +2,29 @@
 from django.contrib.auth.models import User
 from tastypie.utils.timezone import now
 
+class Group(models.Model):
+  users = models.ManyToManyField(User, related_name='users')
```

Comment:
`related_name` is wrong here. It could be `groups` or something. Basically it's better to not use related_name since you already have access to such attributes through django - https://docs.djangoproject.com/en/1.8/topics/db/queries/#related-objects